main.yml

```diff
@@ -0,0 +1,42 @@
+name: build
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        go: [3.2, 3.3, 3.4, 3.5]
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          submodules: recursive
+      - name: Use H3 ${{ matrix.GO }}
```

Comment:
It looks to me like this would only run the checkout in the `test` job but not the tests - are these being run in `report-generation`?

Reply:
Thanks I think I need to write this in github/worklfows as suggested @dfellis ,@isaacbrodsky  can we comfirm whether this test is run in report-generation

Reply:
@isaacbrodsky is right! the commands you're looking for for this job is:
```
cd v3 && go test
```